### PR TITLE
feat: import package linux-show-player from AUR

### PR DESCRIPTION
**Pull request by SpotlightKid**

### CODE REVIEW

**Comment by SpotlightKid** [comment]
Note: before merging, first need to import `python-mido` as well.

**Comment by SpotlightKid** [comment]
I'm only a co-maintainert of the [AUR package](https://aur.archlinux.org/packages/linux-show-player), I can't add `osamc` as a co-maintainer. So sync to AUR is not enabled on purpose and pkgrel is 1, same as in AUR.

If and when a new release comes out, I can contact the AUR maintainer and ask him to add `osamc` or disown the package so I can become maintainer.

File: nvchecker/archlinux-proaudio.toml
Comment (cbix) [inline nvchecker/archlinux-proaudio.toml:85]: This looks wrong 🤔

File: packages/linux-show-player/PKGBUILD
Comment (cbix) [inline packages/linux-show-player/PKGBUILD:29]: It may be worth mentioning that the [Arch Wiki recommends](https://wiki.archlinux.org/title/Python_package_guidelines#Standards_based_(PEP_517)) packaging python packages using a new method, unless that doesn't work for some reason. I don't have much experience with either but it apparently requires adding two dependencies instead of one :D

**Review by cbix — CHANGES_REQUESTED** [review]
Nice, will try testing this later :)

**Review by cbix — APPROVED** [review]

**Pull request merged**
